.github/ISSUE_TEMPLATE/ticket.md

```diff
@@ -50,3 +30,12 @@ about: Defines an item of work for the project
 - [ ] Item A
 - [ ] Item B
```

Comment:
Delete Item B and C. We just need Item A as template

Reply:
I like having a few, as it more clearly indicates it's a series. Definitely a subjective thing though, so perhaps other people can weigh in here?

Reply:
I kinda like the multiple examples. There's generally at least 2 items with each ticket (do the thing and test it) and IMO it's easier to remove them than add them.

Reply:
Ok, just gonna leave it as-is for now then.